packages/mui-material/src/Chip/Chip.js

```diff
@@ -140,6 +140,10 @@ const ChipRoot = styled('div', {
         ...(ownerState.color !== 'default' && {
           color: 'inherit',
         }),
+        ...(ownerState.iconColor &&
+          ownerState.iconColor !== ownerState.color && {
+            color: undefined, // Don't override color defined by icon
```

Comment:
I dont know if the comment is necessary
```suggestion
            color: undefined, 
```

Reply:
Removed